snapcraft/tests/plugins/test_kernel.py

```diff
@@ -863,15 +863,15 @@ def test_kernel_image_target_as_map(self):
         plugin = kernel.KernelPlugin('test-part', self.options,
                                      project_options)
 
-        self.assertEqual(plugin.make_targets, ['Image', 'modules'])
+        self.assertEqual(plugin.make_targets, ['Image', 'modules', 'dtbs'])
```

Comment:
Yes we do although not directly visible on https://github.com/snapcore/snapcraft/pull/1148/files#diff-6f5410d7a6a3500fab904417fc36123dR779

We should probably refactor the kernel tests into many smaller classes but Paolo shouldn't be the one to do that.